### ISSUE TITLE
systemd-sysusers does not create root and nobody users

### ISSUE THREAD

**Issue by seblu** [systemd/systemd#6808]
### Submission type
 - Bug report

### systemd version the issue has been seen with

> systemd 234
+PAM -AUDIT -SELINUX -IMA -APPARMOR +SMACK -SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT +GNUTLS +ACL +XZ +LZ4 +SECCOMP +BLKID +ELFUTILS +KMOD -IDN2 +IDN default-hierarchy=hybrid

### Used distribution

> Arch Linux

### Bug report description

It appears that the root/nobody user/group are not created by systemd-sysusers, despite its definition in sysusers.d/basic.conf.
I guess nss_systemd is always providing answers for root/nobody user/group which mislead systemd-sysusers.

```
# grep nobody /etc/passwd; echo $?
1
# getent passwd nobody
nobody:*:65534:65534:User Nobody:/:/sbin/nologin
# SYSTEMD_LOG_LEVEL=debug systemd-sysusers /usr/lib/sysusers.d/basic.conf
Group nobody already exists.
User nobody already exists.
```
For reference, this was firstly reported in the Arch Linux bug report [FS#55589](https://bugs.archlinux.org/task/55589).

Side effect, sshd doesn't like the value returned by nss_systemd and segfault. More detail in [FS#55570](https://bugs.archlinux.org/task/55570).

I tend to think that systemd-users should be able to start working with and empty passwd/group. Am I correct?

Cheers,

**Comment by loqs** [comment]
The /etc/nsswitch.conf for the issue
```# Name Service Switch configuration file.
# See nsswitch.conf(5) for details.

passwd: files mymachines systemd
group: files mymachines systemd
shadow: files

publickey: files

hosts: files mymachines resolve [!UNAVAIL=return] dns myhostname
networks: files

protocols: files
services: files
ethers: files
rpc: files

netgroup: files
```

**Comment by poettering** [comment]
Fix waiting in #6812
